package.json

```diff
@@ -6,19 +6,19 @@
     "snyk-prevent-gh-commit-status": "dist/index.js"
   },
   "scripts": {
-    "format:check": "prettier --check '{''{lib,test}/!(fixtures)/**/*,*}.{js,ts,json,yml}'",
-    "format": "prettier --write '{''{lib,test}/!(fixtures)/**/*,*}.{js,ts,json,yml}'",
+    "format:check": "prettier --check '{''{lib}/!(fixtures)/**/*,*}.{js,ts,json,yml}'",
+    "format": "prettier --write '{''{lib}/!(fixtures)/**/*,*}.{js,ts,json,yml}'",
     "lint": "npm run format:check && npm run lint:eslint",
     "lint:eslint": "eslint --cache '{lib,test}/**/*.ts'",
-    "test": "snyk test && npm run lint && npm run test:unit",
+    "test": "npm run lint && npm run test:unit",
     "test:unit": "jest",
     "test:coverage": "npm run test:unit -- --coverage",
     "test:watch": "tsc-watch --onSuccess 'npm run test:unit'",
     "build": "tsc",
     "build-watch": "tsc -w",
     "prepare": "npm run build",
     "snyk-test": "snyk test",
-    "pkg-binaries": "npx pkg . -t node12-linux-x64,node12-macos-x64,node12-win-x64 --out-path ./dist/binaries"
+    "pkg-binaries": "npx pkg . -t node18-linux-x64,node18-macos-x64,node18-win-x64 --out-path ./dist/binaries"
```

Comment:
this will potentially break cli comparability, we support Node 12 so binaries need to keep working on node 12. Are the tree sets in circle still running on 12 so we will catch if something stops working?

Reply:
set to node16